ecs-agent/daemonimages/csidriver/go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/aws/amazon-ecs-agent/ecs-agent/daemonimages/csidriver
```

Comment:
IIUC the ecs volume plugin reuses the vendor directory of it's parent directory (ecs-init). Would it be possible to do that here so we don't have another independent vendor/ tree to maintain? I believe this would be the fourth in this repo.

Reply:
+1, not sure why we have an extra go.mod here? Can't we just have it in ecs-agent's go.mod?